packages/types/src/primitive/Generic/Vote.ts

```diff
@@ -4,12 +4,18 @@
 
 import { Conviction } from '../../interfaces/democracy';
 
-import { isBoolean, isNumber, isObject, isU8a, isUndefined } from '@polkadot/util';
+import { isBoolean, isNumber, isU8a, isUndefined } from '@polkadot/util';
 
 import { createType } from '../../codec/create';
 import U8aFixed from '../../codec/U8aFixed';
 import Bool from '../Bool';
 
+// eslint-disable-next-line @typescript-eslint/ban-types
+type InputTypes = boolean | number | Boolean | Uint8Array | {
+  aye: boolean;
+  conviction?: number | ('None' | 'Locked1x' | 'Locked2x' | 'Locked3x' | 'Locked4x' | 'Locked5x');
```

Comment:
out of curiosity, why the parenthesis here?

Reply:
lol, for my sanity - wanted to group the "string values" since they belong together. (And didn't want to define yet another type that is not actually exported.)

To be brutally honest, I'm still rally unhappy with Vote as it stands. This is the 2nd round of cleanups here and it still just doesn't quite get clean. (But each bit helps)